grails-app/conf/UrlMappings.groovy

```diff
@@ -66,7 +66,10 @@ class UrlMappings {
 		"/api/v1/users/me/keys"(resources: "keyApi", excludes: ["create", "edit", "update"]) { resourceClass = SecUser }
 		"/api/v1/users/me/products"(controller: "productApi", action: "index") { operation = Permission.Operation.SHARE }
 
-		"/api/v1/integration_keys"(resources: "integrationKeyApi")
+		"/api/v1/integration_keys"(controller: "integrationKeyApi", action: "index", excludes: ["delete", "update", "save"], method: "GET")
```

Comment:
What does the `excludes` list accomplish here? Seems like a pain to maintain. I thought it is intended to be used with the `resources` parameter (Grails auto-generated mappings).

Reply:
It's the grails magic that is annoying. Now you can see what happens with a single glance.

Reply:
I think you shouldn't need `excludes` here since you are mapping only a single API endpoint. Or is it not working properly without it?

Reply:
Removing grails magic is ok. But the `excludes` statement is only needed when you **use** that grails magic. Now you don't need it!

Reply:
Ok, sounds good!